IntelFsp2Pkg/Library/BaseFspCommonLib/FspCommonLib.c

```diff
@@ -90,8 +90,18 @@ GetFspGlobalDataPointer (
   )
 {
   FSP_GLOBAL_DATA  *FspData;
-
+  UINTN            FspDataAddress;
+ 
   FspData = *(FSP_GLOBAL_DATA  **)(UINTN)PcdGet32 (PcdGlobalDataPointerAddress);

```

Comment:
If we just cast the data during assignment, will that work the same?

 FspData = **(UINT32)** *(FSP_GLOBAL_DATA  **)(UINTN)PcdGet32 (PcdGlobalDataPointerAddress);

Reply:
If change as this code, it will show build error " warning C4311: 'type cast': pointer truncation from 'FSP_GLOBAL_DATA *' to 'UINT32' "

Reply:
I think below should resolve the warning, please help to double check:
FspData = **(UINT32) (UINTN)** *(FSP_GLOBAL_DATA **)(UINTN)PcdGet32 (PcdGlobalDataPointerAddress);